sw/host/tests/manuf/cp_provision_init/src/main.rs

```diff
@@ -90,24 +93,36 @@ fn load_and_run_sram_program(opts: &Opts, transport: &TransportWrapper) -> Resul
     uart.clear_rx_buffer()?;
 
     // Load and execute the SRAM program that contains the provisioning code.
-    opts.sram_program
-        .load_and_execute(&jtag, ExecutionMode::Jump)?;
+    match opts
+        .sram_program
+        .load_and_execute(&jtag, ExecutionMode::Jump)?
+    {
+        ExecutionResult::Executing => {
+            log::info!("SRAM program loaded and is executing.")
+        }
+        res => log::info!("SRAM program load/execution failed: {:?}.", res),
```

Comment:
Do we want to return an error or panic here?

Reply:
ah yeah, good idea. fixed.